sequencer/src/context.rs

```diff
@@ -146,7 +145,12 @@ impl<N: ConnectedNetwork<PubKey>, P: SequencerPersistence, V: Versions> Sequence
             initializer,
             ConsensusMetricsValue::new(metrics),
             Arc::clone(&persistence),
-            marketplace_config,
+            // TODO: MA: will be removed when more marketplace code is removed,
+            // at the moment we need to pass in a config to hotshot.
+            MarketplaceConfig {
```

Comment:
Wondering if this will break anything. I'm thinking if the marketplace feature isn't turned on it shouldn't but not sure.